collector/Makefile

```diff
@@ -48,12 +48,12 @@ package: build
 
 .PHONY: publish
 publish:
-	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/collector-extension-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
+	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/opentelemetry-collector-layer-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
 
 .PHONY: publish-layer
 publish-layer: package
 	@echo Publishing collector extension layer...
-	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/collector-extension-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
+	aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/opentelemetry-collector-layer-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
```

Comment:
I found `collector-extension-$(GOARCH).zip` elsewhere in this repo:

```
tigre@hilleman:~/github/ocelotl/opentelemetry-lambda$ ack collector-extension
go/integration-tests/aws-sdk/wrapper/main.tf
4:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
7:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

python/integration-tests/aws-sdk/wrapper/main.tf
12:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
15:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

nodejs/integration-tests/aws-sdk/wrapper/main.tf
12:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
15:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

collector/Makefile
51:     aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/collector-extension-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text
56:     aws lambda publish-layer-version --layer-name $(LAYER_NAME) --zip-file fileb://$(BUILD_SPACE)/collector-extension-$(GOARCH).zip --compatible-runtimes nodejs14.x nodejs16.x nodejs18.x java11 python3.8 python3.9 --query 'LayerVersionArn' --output text

java/integration-tests/okhttp/wrapper/main.tf
12:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
15:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

java/integration-tests/aws-sdk/wrapper/main.tf
12:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
15:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

java/integration-tests/aws-sdk/agent/main.tf
12:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
15:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

dotnet/integration-tests/aws-sdk/wrapper/main.tf
4:  filename            = "${path.module}/../../../../collector/build/collector-extension-amd64.zip"
7:  source_code_hash    = filebase64sha256("${path.module}/../../../../collector/build/collector-extension-amd64.zip")

utils/sam/run.sh
95:             rm -f build/collector-extension-amd64.zip
```

do we need to also change those other occurrences?

Reply:
The rest of the occurrences in the `.tf` files are invalid for use in this repo. I am going to PR soon to remove those unused files in this repo.